.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Exclude files from exporting
```

Comment:
These `export-ignore` entries are used by `git archive` (part of the GH action flow) to create the zip file.  So you can add other entries if you don't want them included in the zip file